### DISCUSSION TITLE
Multiple Identical VMs

### DISCUSSION THREAD

**Discussion by kevinisageek** [bpg/terraform-provider-proxmox#735] [Q&A] [answered]
Hey folks,

Trying out this provider after the Telmate one has been giving me some issues. Seems to be working so far with one exception.

With the Telmate provider, there was the ability to create multiple identical virtual machines via the ``count`` parameter, is there a way to do this with this provider? Context being a cluster of kubernetes nodes, don't want to have to declare every single one when the only difference between them is the name.

**Comment by zimmertr** [comment, marked as answer]
[Count](https://developer.hashicorp.com/terraform/language/meta-arguments/count#the-count-object) is a meta-argument in Terraform and wasn't just implemented by the Telmate provider. It's used the same way with this (and other) providers. See [here](https://github.com/zimmertr/TJs-Kubernetes-Service/blob/main/talos_controlplanes.tf#L2) for an example.

**Reply by kevinisageek** [reply]
Legend!
